python/pyspark/serializers.py

```diff
@@ -289,7 +291,6 @@ def _create_batch(self, series):
         def create_array(s, t):
             mask = s.isnull()
```

Comment:
Do we still need to use `mask`?

Reply:
I'm not sure, I'll check

Reply:
Yes, it's need to correctly insert NULL values in timestamps, since there is a `fillna(0)` done on the series.